src/standard/utils.html

```diff
@@ -219,14 +219,14 @@
      * property.
      *
      * @method translate3d
-     * @param {HTMLElement} node Element to apply the transform to.
      * @param {number} x X offset.
      * @param {number} y Y offset.
      * @param {number} z Z offset.
+     * @param {HTMLElement} node Optional Element to apply the transform to.
```

Comment:
For optional arguments, you can call that out via the type expression:

    @param {HTMLElement=} node Element to apply the transform to.